abdm_python_integrator/abha/utils/response_util.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_500_INTERNAL_SERVER_ERROR
+
+success_response_keys = ["txnId", "healthIdNumber", "auth_methods", "token",
+                         "health_card", "health_id", "authMethods"]
+
+
+def parse_response(response_data):
+    if not response_data:
+        return Response({"error": "No valid response found"}, status=HTTP_500_INTERNAL_SERVER_ERROR)
+    if any([key in response_data for key in success_response_keys]):
+        return _get_success_abdm_response(response_data)
+    return _get_error_abdm_response(response_data)
+
+
+def generate_invalid_req_response(message, error_code=HTTP_400_BAD_REQUEST):
```

Comment:
Could call this `get_ bad_response` to be consistent with other method names.

Reply:
Addressed in 6e8657a68d991b46a3eaa5d5901c4f254af0062c